etc/docker/kubernetes-kernel-tf-py-gpu/bootstrap-kernel.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+echo kernel-bootstrap.sh: language=${KERNEL_LANGUAGE}, connection-file=${KERNEL_CONNECTION_FILENAME}, reponse-addr=${EG_RESPONSE_ADDRESS}, spark-context-init-mode=${KERNEL_SPARK_CONTEXT_INIT_MODE}
+
+if [[ "${KERNEL_LANGUAGE}" == "python" ]];
+then
+	echo "python /usr/local/share/jupyter/kernels/python_kubernetes/scripts/launch_ipykernel.py ${KERNEL_CONNECTION_FILENAME} --RemoteProcessProxy.response-address ${EG_RESPONSE_ADDRESS} --RemoteProcessProxy.spark-context-initialization-mode ${KERNEL_SPARK_CONTEXT_INIT_MODE}"
+	python /usr/local/share/jupyter/kernels/python_kubernetes/scripts/launch_ipykernel.py ${KERNEL_CONNECTION_FILENAME} --RemoteProcessProxy.response-address ${EG_RESPONSE_ADDRESS} --RemoteProcessProxy.spark-context-initialization-mode ${KERNEL_SPARK_CONTEXT_INIT_MODE}
```

Comment:
Thanks for the contribution @atinsood.  I just realized there's an issue in the path for the `launch_ipykernel.py` file.  This issue also exists in the the `python_tf_kubernetes` bootstrap as well (from which this one was derived).  Could you please submit a fix for this file as well as `python_tf_kubernetes`?

The path referenced in both lines should be `/usr/local/share/juptyer/kernels/python_tf_kubernetes/scripts/launch_ipykernel.py` and then `/usr/local/share/juptyer/kernels/python_tf_gpu_kubernetes/scripts/launch_ipykernel.py`...

It looks like there isn't a `kernelspecs` directory for this kernel where the `kernel.json` file resides, so I would recommend adding a new directory to the kernelspecs hierarchy as well.  Dang, that same path issue exists in the `python_tf_kubernetes/kernel.json` file as well!  (So if you could please fix that with your update, it would be much appreciated!)

Reply:
@kevin-bates done

Reply:
Thank you.